openai/init.moon

```diff
@@ -366,7 +364,6 @@ class OpenAI
       cjson.encode payload
 
     headers = {
-      "Host": parse_url(@api_base).host
```

Comment:
Host header is part of HTTP spec, not related to any of the APIs. As this library tries to support many environments/clients, in my experience I've had some issues with Lua http clients not correctly setting host header based on the request URL. For that reason I would not remove it here.

Reply:
Firstly, in my humble opinion, it appears that we are trying to solve a problem that does not yet exist. 

Having tested the LuaSocket with numerous different services without explicitly setting the Host in the library code, I didn’t encounter any problems. Of course, there is always the possibility someone, someday, might encounter an issue in a different environment.

In such an event, we would still have three options:

1. Request a proper fix in the appropriate location (i.e. the custom Lua HTTP client).
2. Instead of waiting for it to be fixed, set the header in the user code with solution #4.
3. As a last resort, we can always modify the Lua-OpenAI code itself, preferably not prematurely. 

However, the final decision is ultimately up to you and I will withdraw the last commit if you insist.